notebooks/features/responsible_ai/Interpretability - Snow Leopard Detection.ipynb

```diff
@@ -233,21 +232,15 @@
     "    return float(row[1])\n",
     "\n",
     "\n",
-    "if running_on_synapse():\n",
-    "    network = ModelDownloader(\n",
-    "        spark, \"abfss://synapse@mmlsparkeuap.dfs.core.windows.net/models/\"\n",
-    "    ).downloadByName(\"ResNet50\")\n",
-    "elif running_on_databricks():\n",
-    "    network = ModelDownloader(spark, \"dbfs:/Models/\").downloadByName(\"ResNet50\")\n",
-    "else:\n",
-    "    network = ModelDownloader(spark, \"/tmp/Models/\").downloadByName(\"ResNet50\")\n",
-    "\n",
     "model = Pipeline(\n",
     "    stages=[\n",
     "        StringIndexer(inputCol=\"labels\", outputCol=\"index\"),\n",
     "        ImageFeaturizer(\n",
-    "            inputCol=\"image\", outputCol=\"features\", cutOutputLayers=1\n",
-    "        ).setModel(network),\n",
+    "            inputCol=\"image\",\n",
+    "            outputCol=\"features\",\n",
+    "            autoConvertToColor=True,\n",
+    "            ignoreDecodingErrors=True,\n",
+    "        ).setModel(\"ResNet50\"),\n",
```

Comment:
later we should figure out how to make this also settable with kwargs (this will probabbly be uglier to implement than it loos so dw about this PR)

Reply:
ok